test/integ/testdata/gen-binding.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2020, Oracle and/or its affiliates.
```

Comment:
since these aren't used, I would suggest taking out the model and binding files

Reply:
fixed